spec/features/snap_app_spec.rb

```diff
@@ -40,7 +40,29 @@
       click_on "Continue"
     end
 
-    click_on "Continue"
+    on_page "Your Household" do
+      click_on "Continue"
+    end
+
+    on_page "Money & Income" do
+      choose "Yes"
+      click_on "Continue"
+    end
+
+    on_page "Money & Income" do
+      fill_in "step[income_change_explanation]", with: "EXPLAINING!!!"
+      click_on "Continue"
+    end
+
+    # this is next
+    # on_page "Money & Income" do
+    #   within(find(".household-member-group", text: "Jessie (that's you!)")) do
+    #     choose("Employed")
+    #   end
+    #   within(find(".household-member-group", text: "Joey")) do
+    #     choose("Not Employed")
+    #   end
+    # end
```

Comment:
sure -- was just saving for future me :) (that's the next thing I am working on)